### ISSUE TITLE
Unable to install playwright-crx through npm

### ISSUE THREAD

**Issue by molvqingtai** [ruifigueira/playwright-crx#16]
When I ran `pnpm i playwright-crx`, the installation failed and no file was found: `playwright/packages/playwright/bundles/babel`

This seems to be used because the local file path is used in package.json. Why not set playwright to peer dependencies?

**Comment by ruifigueira** [comment]
You're right, with `pnpm` it's not working. With `npm` it is working, that's why I never noticed that issue.

I'm using those local playwright bundles dependencies to avoid declaring all their transitive dependencies explicitly. Can you use `npm` instead to install it?

**Comment by ducan-ne** [comment]
@ruifigueira use npm works and it is fine for testing, but do you have plan to make pnpm works? For production I (and people) usually use pnpm

**Comment by ruifigueira** [comment]
8f06b95fa3527bf72ceaff6ad103d67e7f65fd53 should fix the issue. You can try it:

```
git clone git@github.com:ruifigueira/playwright-crx.git
npm ci
npm build
npm pack
```
This will generate a `playwright-crx-0.6.0-next.tgz`.

Then create a new temp folder, copy the `playwright-crx-0.6.0-next.tgz`. file there and run:

```
pnpm add playwright-crx-0.6.0-next.tgz
```

**Comment by ruifigueira** [comment]
I released [playwright-crx@0.6.0](https://www.npmjs.com/package/playwright-crx), it now works well with `pnpm`.
